packages/expo-av/src/WebFullscreenUtils.ts

```diff
@@ -0,0 +1,105 @@
+const supportsFullscreenAPI = (element: HTMLMediaElement): boolean =>
+  'requestFullscreen' in element;
+
+const supportsWebkitFullscreenAPI = (element: HTMLMediaElement): boolean =>
+  'webkitEnterFullScreen' in element;
+
+const supportsWebkitFullscreenChangeEvent = (): boolean => {
+  const element = document.createElement('video');
+  element.setAttribute('onwebkitfullscreenchange', 'return;');
+  return typeof element['onwebkitfullscreenchange'] === 'function';
+};
```

Comment:
Please add comments or a code reference (stack overflow article?) on why this works and for which browsers